yang/frr-nexthop.yang

```diff
@@ -200,10 +204,45 @@ module frr-nexthop {
           "Identifies a nexthop within a nexthop group; the entries
            are ordered by id value, and the value has no other meaning.";
```

Comment:
again, the id value of a nexthop does have meaning now, in the relationships among individual nexthops and groups zebra composes them currently.